gapic.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+CMD="$0"
+
+# Set variables used by this script.
+# All of these are set in options below, and all but $PATH are required.
+GO_GAPIC_OPT=
+IMAGE=
+IN=
+OUT=
+PROTO_PATH=`pwd`
+
+# Print help and exit.
+function show_help {
+  echo "Usage: $CMD --go_gapic_opt GO_GAPIC_OPT --image IMAGE --in IN_DIR --out OUT_DIR [--path PATH_DIR]"
+  echo ""
+  echo "Required arguments:"
+  echo "      --go_gapic_opt  The import path and name for the generated package"
```

Comment:
It still need not block this PR, but let's have a meeting to discuss this. Go requires this option, but other languages likely need to permit it.